app/models/user.rb

```diff
@@ -1,4 +1,9 @@
 class User < ApplicationRecord
+  attribute :display_name
```

Comment:
I'm not sure that you need it in here, because this method will exist anyway. Read more what `attribute` method is used for